angular-ngc/defs.bzl

```diff
@@ -35,21 +37,26 @@ TEST_CONFIG = [
 ]
 TEST_DEPS = APPLICATION_DEPS + [
     "//:node_modules/@angular/compiler",
+    "//:node_modules/@angular/platform-browser-dynamic",
     "//:node_modules/@types/jasmine",
     "//:node_modules/jasmine-core",
+    "//:node_modules/karma-chrome-launcher",
```

Comment:
We should separate the dependencies into those from the test code (spec files) and a separate list for the test-runner (karma stuff).

Reply:
Yes, you are right, I updated this one in my local